### DISCUSSION TITLE
pvc & secrets

### DISCUSSION THREAD

**Discussion by mybigopenbox** [signalfx/splunk-otel-collector#2135] [Q&A] [answered]
Hi

I am using this chart and it works great.
However, I wan to expand the information that is collected.

Want I would like to know is how to I get information from the collectors about K8s PVCs and kK8s secrets so I can display the information in a splunk dashboard

**Comment by atoulme** [comment, marked as answer]
You should be able to get this information with this receiver: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/documentation.md
